sdk/rust/oak/src/lib.rs

```diff
@@ -298,11 +299,19 @@ pub fn channel_write(half: WriteHandle, buf: &[u8], handles: &[Handle]) -> Resul
     result_from_status(status as i32, ())
 }
 
-/// Create a new unidirectional channel.
+/// Create a new unidirectional channel with the specified label.
+///
+/// Only nodes with a compatible label will be able to read and write to this channel:
+///
+/// - A node with label `L_w` may write to a channel with label `L_c` iff `L_w ⊑ L_c`.
+/// - A node with label `L_r` may read from a channel with label `L_c` iff `L_c ⊑ L_r`.
```

Comment:
In a future scenario where nodes are multi-threaded this could be tricky. One thread might try to raise the node's label to read a new message while another might lower the same node's label to write a declassified message. This would potentially require locks spanning large sections of code, effectively making it single-threaded again. A mechanism where a node could declassify a single message without changing its overall label seems easier to use.